docs/content/en/docs/getting-started/production-environment/baremetal-getstarted.md

```diff
@@ -66,8 +66,10 @@ Follow these steps to create an EKS Anywhere cluster.
      
 1. Create the cluster, using the `hardware.csv` file you made in [Bare Metal preparation]({{< relref "/docs/reference/baremetal/bare-preparation.md" >}}):
    ```bash
-   # Create a cluster without curated packages installation
-   eksctl anywhere create cluster --hardware-csv hardware.csv -f eksa-mgmt-cluster.yaml
+   eksctl anywhere create cluster \
+      --hardware-csv hardware.csv \
+      -f eksa-mgmt-cluster.yaml  \
```

Comment:
As it stands, the command won't work because you end the uncommented text with a backslash. How about:

      -f eksa-mgmt-cluster.yaml  # --install-packages packages.yaml
       # uncomment above to install curated packages at cluster creation